cd_database/migrations/1714995407000000_event_sourcing_light.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS event_sourcing_light -- aka ESL
+(
+    created TIMESTAMP,
+    event_type VARCHAR(255),
+    json VARCHAR,
+    PRIMARY KEY(created)
```

Comment:
`PRIMARY KEY(created)` Hmm, this means we can only have one event per tick and never a series of events in a batch. Are we sure we'll never need that? I'd prefer a simple increment key, because then we could also rep. ordered events on the same timestamp.

Reply:
very good point! I added a proper primary key. Decided to go for UUID.